.gitlab/functional_test/serverless.yml

```diff
@@ -1,15 +1,14 @@
 serverless_cold_start_performance-deb_x64:
```

Comment:
Side note: the serverless-agent team should own that file. They don't have a Github group however (I can only see `serverless`, which likely contains too many people).